virttest/qemu_vm.py

```diff
@@ -996,6 +996,22 @@ def add_memorys(devices, params):
                 for name in params.objects("mem_devs"):
                     dev = devices.memory_define_by_params(params, name)
                     devs.extend(dev)
+            machine_dev = devices.get_by_properties({"type": "machine"})[0]
+            machine_cmd = machine_dev.cmdline_nd()
+            output = re.findall(r",memory-backend=mem-([\w|-]+)", machine_cmd)
+            if output:
+                name = output[0]
+                params["size_%s" % name] = "%sM" % params["mem"]
+                params["use_mem_%s" % name] = "no"
```

Comment:
For this memory object for machine, can `use_mem` only be `no` ?

Reply:
Yes, it must be 'no', otherwise, a dimm device will generate.

Reply:
ok, thanks!